package.json

```diff
@@ -25,7 +25,7 @@
   },
   "scripts": {
     "start": "node server.js",
-    "start:devserver": "npx cross-env NODE_ENV=testing nodemon server.js --watch ./server --watch ./server.js",
+    "start:devserver": "concurrently \"nodemon ./server/proxy/server.js\" \"npx cross-env NODE_ENV=testing nodemon --inspect server.js --watch ./server --watch ./server.js\"",
```

Comment:
What are you trying to do here?

Reply:
this was from sisi's method of doing two browsers

Reply:
Ah okay, in the latest changes to develop there is now a `start:testserver` script so this will probably be changed soon, no worries